install/unbound-install.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: wimb0
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  sudo \
+  curl \
+  mc 
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Unbound"
+$STD apt-get install -y \
+  unbound \
+  unbound-host
+msg_info "Installed Unbound"
+
+cat <<EOF >/etc/unbound/unbound.conf.d/unbound-lxe.conf
+server:
+  verbosity: 0
```

Comment:
```suggestion
  verbosity: 1
```
What is this configuration file based on? It should be a minimal working configuration with mostly defaults

Reply:
I based this config on the configuration from Pi-hole (https://docs.pi-hole.net/guides/dns/unbound/) and used parts of https://github.com/saint-lascivious/unbound-config